### DISCUSSION TITLE
Remote to my real browser instead of external chromedriver

### DISCUSSION THREAD

**Discussion by foremtehan** [php-webdriver/php-webdriver#858] [Q&A] [answered]
Hello folks, I want to know is it possible to connect my installed browser instead of external chromedriver  (in windows) ? In python i can do it like:

```python
import subprocess

subprocess.Popen(['C:/Program Files (x86)/Google/Chrome/Application/chrome.exe', '--remote-debugging-port=7102', 'url'])

options = Options()
options.add_experimental_option("debuggerAddress", "localhost:7102")
driver = webdriver.Chrome(options=options)
//now i can able to control my real browser
```

Is it possible to simulate this in php with this package?

**Comment by OndraM** [comment, marked as answer]
Hi, have look at [Chrome](https://github.com/php-webdriver/php-webdriver/wiki/Chrome) page in our wiki, especially sections "Installation" and "Start directly using ChromeDriver class". 

Beware you also need to download `chromedriver` (`chromedriver.exe`) binary (see the wiki for details), having just chrome.exe isn't enough.

**Comment by jskorlol** [comment]
Is it currently not possible to connect to external chrome in php?

**Reply by OndraM** [reply]
Hi, I'm not sure, what exactly you mean, but have a look at [our wiki](https://github.com/php-webdriver/php-webdriver/wiki/Chrome#start-chromedriver). It is possible to connect to either local chromedriver or any remote end, including remote chrome via selenium server.

**Reply by jskorlol** [reply]
Oh, it worked. thanks !
